core/templates/base-components/oppia-root.directive.ts

```diff
@@ -31,6 +33,15 @@ angular.module('oppia').directive('oppiaRoot', [
           $scope.initialized = false;
 
           $scope.onInit = function() {
+            const translateService = (
```

Comment:
In the old structure I see `translateService.use('en');` in the constructor, do we need this anywhere?

Reply:
No, not required, since the default value in the translate service is 'en'.